### PR TITLE
Add tx_type to request fields

### LINKED ISSUE #2137: Update doc for account_tx wrt tx_type field
See https://github.com/XRPLF/clio/pull/851 for the changes.

### PR DESCRIPTION
**Pull request by godexsoft**
Fixes #2137 

I don't think an example is needed but feel free to let me know if we should add one as well as any changes needed to the line that i added in this PR.

### CODE REVIEW

File: content/references/http-websocket-apis/public-api-methods/account-methods/account_tx.md
Comment (intelliot) [inline content/references/http-websocket-apis/public-api-methods/account-methods/account_tx.md]: In addition to mentioning what is **not** supported (AMM*), I think it may be helpful to list some examples of `tx_type` strings which **are** valid, something like:

```suggestion
| `tx_type`          | String                                     | _(Optional)_ **Clio Only** Return only transactions of a specific type, such as "Clawback", "AccountSet", "AccountDelete", et al. Case-insensitive. Supports any transaction type except `AMM*` (See [Transaction Types](transaction-types.html).) [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
```

**Comment by ckniffen** [comment]
Does it make sense to document these features before they are released?  Maybe a list the version it is added in.

**Comment by godexsoft** [comment]
> Does it make sense to document these features before they are released? Maybe a list the version it is added in.

We did add the `[New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE")` at the end.
I agree we can hold off merging/publishing this until we are ready with the release tho.

**Comment by ckniffen** [comment]
Woops sorry i missed that.

**Comment by ckniffen** [comment]
I think it is fine with the badge as people might encounter it in the wild still through s2

**Review by intelliot — APPROVED** [review]
LGTM

**Comment by mDuo13** [comment]
I'd prefer to wait until Clio v2 is properly released to merge & deploy these changes, especially since the new fields added are on requests, not responses, so users probably won't encounter them "in the wild" without knowing about them.

**Review by mDuo13 — APPROVED** [review]

**Pull request merged**
